codespell_lib/data/dictionary.txt

```diff
@@ -53203,6 +53357,7 @@ tuotiral->tutorial
 tuotirals->tutorials
 tupe->tuple, type,
 tupel->tuple
```

Comment:
`tupelo` and plural are probably uncommon, aren't they?

Reply:
I couldn't find this typo.